src/foreman_mcp_server/utils/dsl_docs_utils.py

```diff
@@ -1,220 +1,242 @@
 
-  output_lines = [header, "", "## ERB Templates"]
+    output_lines = [header, "", "## ERB Templates"]
 
-  # These are not part of the DSL documentation
-  erb_templates = [
-    ("<% code %>", "Executes code, but does not insert a value"),
-    ("<%= expression %>", "Inserts the value of an expression"),
-    ("<% code -%>", "Executes code, but does not insert a value, trims the following line break"),
-    ("<%= expression -%>", "Inserts the value of an expression, trims the following line break"),
-    ("<%# comment -%>", "Comment, removed from the final output, trims the following line break"),
-  ]
+    # These are not part of the DSL documentation
+    erb_templates = [
+        ("<% code %>", "Executes code, but does not insert a value"),
+        ("<%= expression %>", "Inserts the value of an expression"),
+        (
+            "<% code -%>",
+            "Executes code, but does not insert a value, trims the following line break",
+        ),
+        (
+            "<%= expression -%>",
+            "Inserts the value of an expression, trims the following line break",
+        ),
+        (
+            "<%# comment -%>",
+            "Comment, removed from the final output, trims the following line break",
+        ),
+    ]
 
-  for snippet, description in erb_templates:
-    output_lines.append(f"- `{snippet}` - {description}")
+    for snippet, description in erb_templates:
+        output_lines.append(f"- `{snippet}` - {description}")
 
-  output_lines.extend(["", "## DSL Macros", ""])
+    output_lines.extend(["", "## DSL Macros", ""])
 
-  try:
-    cache_data = json.loads(cache_content)
-    classes = cache_data.get('docs', {}).get('classes', {})
+    try:
+        cache_data = json.loads(cache_content)
+        classes = cache_data.get("docs", {}).get("classes", {})
 
-    for class_name, class_desc in classes.items():
-      output_lines.extend(_format_class_documentation(class_name, class_desc))
+        for class_name, class_desc in classes.items():
+            output_lines.extend(_format_class_documentation(class_name, class_desc))
 
-  except (json.JSONDecodeError, KeyError) as e:
-    output_lines.extend(["", f"Error parsing documentation: {str(e)}", ""])
+    except (json.JSONDecodeError, KeyError) as e:
+        output_lines.extend(["", f"Error parsing documentation: {str(e)}", ""])
 
-  return '\n'.join(output_lines)
+    return "\n".join(output_lines)
 
 
 def get_dsldocs_path(cache_dir, cache_file) -> str:
-  """Returns the path to the DSL documentation cache file."""
+    """Returns the path to the DSL documentation cache file."""
 
-  dsldoc_cache_dir = os.path.join(cache_dir, 'dsldoc')
-  if not os.path.exists(dsldoc_cache_dir):
-    os.makedirs(dsldoc_cache_dir, exist_ok=True)
+    dsldoc_cache_dir = Path(cache_dir) / "dsldoc"
+    if not dsldoc_cache_dir.exists():
+        dsldoc_cache_dir.mkdir(parents=True, exist_ok=True)
+
+    return str(dsldoc_cache_dir / cache_file)
```

Comment:
I don't think we "in rome" anymore :D

Reply:
Actually, welcome to the odd district of rome, where division on Path is concatenation with a platform-specific separator

```
>>> from pathlib import Path

>>> Path("asd") / "qwe"
PosixPath('asd/qwe')

>>> str(Path("asd") / "qwe")
'asd/qwe'
```